lookup/simple-lookup-service-server/src/test/java/net/es/lookup/resources/MainResourceTest.java

```diff
@@ -0,0 +1,142 @@
+package net.es.lookup.resources;
+
+import com.google.gson.Gson;
+import net.es.lookup.common.Message;
+import net.es.lookup.common.exception.api.ForbiddenRequestException;
+import net.es.lookup.common.exception.internal.DuplicateEntryException;
+import net.es.lookup.database.ServiceElasticSearch;
+import net.es.lookup.database.connectDB;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.joda.time.DateTime;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+
+public class MainResourceTest {
+
+  private ServiceElasticSearch client;
+
+  private static Logger Log = LogManager.getLogger(RecordResourceTest.class);
+
+  /**
+   * Connects to the database an deletes all records if any exist
+   *
+   * @throws URISyntaxException for incorrect server name
+   * @throws IOException for error in deleting all records
+   */
+  @Before
+  public void setUp() throws URISyntaxException, IOException {
+    client = connectDB.connect();
+    client.deleteAllRecords();
+  }
+
+  /**
+   * closes the connection with the database
+   *
+   * @throws IOException if error in closing connection to the database
+   */
+  @After
+  public void tearDown() throws IOException {
+    client.closeConnection();
+  }
+
+  /**
+   * Curl request to add a record to database that doesn't already exist
+   *
+   * @throws IOException // Error getting result from database
+   * @throws InterruptedException // Sleep interrupted
+   */
+  @Test
+  public void postHandlerNotExist() throws IOException, InterruptedException {
+    MainResource request = new MainResource();
+    request.postHandler("lookup", jsonMessage());
+    Thread.sleep(1000);
+    client.getRecordByURI("lookup/interface/2");
+  }
+
+  /**
+   * Curl request to add a record to database that already exists
+   *
+   * @throws IOException // Error getting result from database
+   * @throws InterruptedException // Sleep interrupted
+   */
+  @Test
+  public void postHandlerExist() throws IOException, InterruptedException {
+    MainResource request = new MainResource();
+    try{
+    request.postHandler("lookup", jsonMessage());
+    Thread.sleep(1000);
+    request.postHandler("lookup", jsonMessage());
+    Log.error("Should have given ForbiddenRequestException");
+    fail();
+    } catch (ForbiddenRequestException e){
+      Log.info("Record already exists, pass");
+    }
+  }
+
+  @Test
+  public void getHandler() {
+    // Todo ??
+  }
+
+  @Test
+  public void bulkRenewHandler() {}
```

Comment:
add a test for bulkrenew

Renew 2 uris - one uri that exists and one that does not.
Database should update the uri that exists and return appropriate messages